packages/vue/src/useHx.ts

```diff
@@ -1,10 +1,10 @@
 import { inject } from "vue";
-import { uberschriftSymbol } from "./uberschriftSymbol";
+import { uberschriftLevel } from "./uberschriftSymbol";
 
 type HeadingTagName = "h1" | "h2" | "h3" | "h4" | "h5" | "h6";
 
 export const useHx = () => {
-	const level = inject<number>(uberschriftSymbol) ?? 1;
```

Comment:
oh wow, i didn't expect that to fail in the tests. Ususally the as the second parameter is a default in case nothing was provided... :/
Now I see why it was like that before 🤷🏻‍♀️ 
Will revert the change